packages/components/src/Tabs/Tabs.tsx

```diff
@@ -12,10 +12,24 @@ import { Typography } from "../Typography";
 
 interface TabsProps {
   readonly children: ReactElement | ReactElement[];
+
+  /**
+   * Specifies the tab that should be active on mount
+   */
+  readonly defaultTab?: number;
+
+  /**
+   * Callback that fires when the active tab changes
+   * @param newTabIndex
+   * @param prevTabIndex
+   */
+  onTabChange?(newTabIndex: number, prevTabIndex: number): void;
 }
 
-export function Tabs({ children }: TabsProps) {
-  const [activeTab, setActiveTab] = useState(0);
+export function Tabs({ children, defaultTab, onTabChange }: TabsProps) {
+  const activeTabInitialValue =
+    defaultTab && defaultTab < React.Children.count(children) ? defaultTab : 0;
```

Comment:
I think we can safely set the defaultTab to 0 if it's not defined

```suggestion
export function Tabs({ children, defaultTab = 0, onTabChange }: TabsProps) {
  const activeTabInitialValue =
    defaultTab < React.Children.count(children) ? defaultTab : 0;
```

that way you don't have to guard it